symfony/plugins/orangehrmCorePlugin/Service/DateTimeHelperService.php

```diff
@@ -131,4 +131,53 @@ public function getNow(DateTimeZone $timezone = null): DateTime
     {
         return new DateTime('now', $timezone);
     }
+
+    /**
+     * @param $baseTimestamp
+     * @return DateTime
+     */
+    public function getDateTimeByTimestamp($baseTimestamp): DateTime
+    {
+        $dateTime = new DateTime();
+        return $dateTime->setTimestamp($baseTimestamp);
+    }
+
+    /**
+     * @param  int  $baseTimestamp
+     * @param  float  $timezoneOffset
+     * @return DateTime
+     */
+    public function getUTCDateTime(int $baseTimestamp, float $timezoneOffset): DateTime
+    {
+        $utcDateTime = new DateTime();
+        return $utcDateTime->setTimestamp($baseTimestamp - $timezoneOffset * 3600);
+    }
+
+    /**
+     * @param  string  $dateTime
+     * @return DateTime
+     */
+    public function getDateTimeByString(string $dateTime): DateTime
+    {
+        return DateTime::createFromFormat('Y-m-d H:i', $dateTime);
+    }
```

Comment:
can't we do this `new \DateTime($dateTimeString);`
e.g. `new \DateTime('2022-02-07 08:08');`